tests/jvm/src/test/scala/scala/meta/tests/semanticdb/SemanticdbSuite.scala

```diff
@@ -114,6 +135,10 @@ abstract class SemanticdbSuite extends FunSuite
     }
   }
 
+  def javaSymbols(code: String, expected: String): Unit = {
```

Comment:
Can we drop the default value for `language`, since we seem to have specialized Java- and Scala-specific methods anyway?

Reply:
You mean drop the default `= s.Language.SCALA` in `checkSection`? Sure, done.

Reply:
👍